// ng2-loading-animate/src/ng2-loading-animate.module.js
var __decorate = (this && this.__decorate) || function (decorators, target, key, desc) {
    var c = arguments.length, r = c < 3 ? target : desc === null ? desc = Object.getOwnPropertyDescriptor(target, key) : desc, d;
    if (typeof Reflect === "object" && typeof Reflect.decorate === "function") r = Reflect.decorate(decorators, target, key, desc);
    else for (var i = decorators.length - 1; i >= 0; i--) if (d = decorators[i]) r = (c < 3 ? d(r) : c > 3 ? d(target, key, r) : d(target, key)) || r;
    return c > 3 && r && Object.defineProperty(target, key, r), r;
};
var __metadata = (this && this.__metadata) || function (k, v) {
    if (typeof Reflect === "object" && typeof Reflect.metadata === "function") return Reflect.metadata(k, v);
};
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { LoadingAnimateComponent } from './ng2-loading-animate.component';
import { LoadingAnimateService } from './ng2-loading-animate.service';
export * from './ng2-loading-animate.service';
export var LoadingAnimateModule = (function () {
    function LoadingAnimateModule() {
    }
    LoadingAnimateModule.forRoot = function () {
        return {
            ngModule: LoadingAnimateModule,
            providers: [LoadingAnimateService]
        };
    };
    LoadingAnimateModule = __decorate([
        NgModule({
            declarations: [
                LoadingAnimateComponent
            ],
            imports: [CommonModule],
            exports: [LoadingAnimateComponent],
            providers: [LoadingAnimateService]
        }), 
        __metadata('design:paramtypes', [])
    ], LoadingAnimateModule);
    return LoadingAnimateModule;
}());
